ui-tests/src/test/java/org/openmrs/reference/AddProviderTest.java

```diff
@@ -26,14 +26,14 @@
     private String personUuid;
 
     @Before
-    public void setUp() {
+	public void setUp() throws Exception {
         person = TestData.generateRandomPerson();
         personUuid = TestData.createPerson(this.person);
     }
 
-    @Ignore
+	@Test
     @Category(BuildTests.class)
-    public void addProviderTest() throws InterruptedException {
```

Comment:
However this  doesnot change anything

Reply:
> However this doesnot change anything

According to method implementation, there other exceptions besides InterruptException that can occur and so  using the BaseClass _Exception_ would be a better approach rather than the child class